flake.nix

```diff
@@ -60,12 +68,90 @@
           '';
         };
 
+        ipfs-sniffer = pkgs.buildGoModule rec {
+          pname = "ipfs-sniffer";
+          version = "master";
+          src = pkgs.fetchFromGitHub {
+            owner = "ipfs-search";
+            repo = pname;
+            rev = "4a7c4441fc9039ce96edffc6ab30d08ac4c2a5f6";
+            sha256 = "sha256-HO21gjQLqzunkbJcZ1Hs+AnaaXJbsiL5BET0MDDWJZ0=";
+          };
+          vendorSha256 = "sha256-xc1biJF4zicosSTFuUv82yvOYpbuY3h++rhvD+5aWNE=";
+        };
+
+        jaeger = pkgs.buildGoModule rec {
+          pname = "jaeger";
+          version = "jaegertracing";
+          src = pkgs.fetchFromGitHub {
+            owner = "jaegertracing";
+            repo = pname;
+            rev = "v1.25.0";
+            sha256 = "sha256-QzHWgjtCKtDVMNkVx82JqsslgIVuCso/xz6ZdQmmkNs=";
+          };
+          vendorSha256 = "sha256-f/DIAw8XWb1osfXAJ/ZKsB0sOmFnJincAQlfVHqElBE=";
+        };
+
+        kibana7-oss = prev.kibana7-oss.overrideAttrs (old: {
```

Comment:
I've upstreamed the update, actually the latest oss version is 7.10.2
If you have a moment, I just need the sha256 for linux https://github.com/NixOS/nixpkgs/pull/136137
(no worries if you are busy).

Reply:
The PR is mostly complete, I'm testing on the server right now. This should take care of kibana7-oss and elasticsearch7-oss

Reply:
I just discovered that the elk stack is not distributed under open source anymore. Those oss versions won't be supported.
I guess we can keep those for now, but probably when nixpkgs gets updated, we'll have to change those.

Reply:
I picked version 7.8.1 of kibana/elasticsearch specifically because it is the one used in the upstream docker-compose.
I don't see good reasons to upgrade packages in nixpkgs just because we need it in this flake. We could introduce other incompatibilities within nixpkgs by updating the package while we're the only ones who really need this version.

Reply:
I didn't see they used a specific version. You're right on this one. Specifically since those versions are end of life already. Those should definitely be inputs to this flake. It makes sense.